ibm/resource_ibm_iam_access_group_members.go

```diff
@@ -98,9 +97,11 @@ func resourceIBMIAMAccessGroupMembersCreate(context context.Context, d *schema.R
 		return diag.FromErr(err)
 	}
 
-	request := prepareMemberAddRequest(userids, serviceids)
+	members := prepareMemberAddRequest(iamAccessGroupsClient, userids, serviceids)
 
-	_, err = iamuumClient.AccessGroupMember().Add(grpID, request)
+	addMembersToAccessGroupOptions := iamAccessGroupsClient.NewAddMembersToAccessGroupOptions(grpID)
+	addMembersToAccessGroupOptions.SetMembers(members)
+	_, _, err = iamAccessGroupsClient.AddMembersToAccessGroup(addMembersToAccessGroupOptions)
 	if err != nil {
 		return diag.FromErr(err)
```

Comment:
return diag.FromErr(err) ===> return diag.FromErr(fmt.Errorf("Error adding members to group(%s). API responce: %s", grpID, detailResponse))